src/styles/blocks/header.scss

```diff
@@ -0,0 +1,82 @@
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  height: 60px;
+
+  &__logo {
+    position: relative;
+    margin-left: 50px;
+    justify-self: flex-start;
+    top: +2px;
+  }
+
+}
+.nav {
+  display: flex;
+  height: 60px;
+
+  &__list {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    margin-right: 50px;
+  }
+  &__item {
+    position: relative;
+    margin: 10px;
+    flex-shrink: 0;
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+  }
+  &__link {
+    cursor: pointer;
+    display: flex;
+    position: relative;
+    padding: 22.5px 0;
+    font-family: "Roboto", sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    text-decoration: none;
+    text-transform: uppercase;
+    line-height: 14px;
+    color: #060b35;
+
+    &:hover {
+      position: relative;
+      color: rgb(0, 172, 220);
+      bottom: -2px;
```

Comment:
Why do you need this value?

Reply:
I tried to pass the test, one of the tests showed a different padding =)